### PR TITLE
Deleted Bower

### PR DESCRIPTION
**Pull request by mknoedel**
Deleting Bower.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,13 +0,0 @@
-{
-  "name" : "FriendTrends",
-  "version" : "0.0.0",
-  "description" : "FriendTrends implementation",
-  "authors" : [
-    "Michael Knoedel",
-    "Navin Pathak"
-  ],
```
Comment (mknoedel) [inline bower.json]: This is my FAVORITE part

File: .bowerrc [hunk 2]
```diff
@@ -1,3 +0,0 @@
-{
-	"directory": "public/lib"
```
Comment (mknoedel) [inline .bowerrc]: Starting a review

Comment (mknoedel) [inline .bowerrc]: Adding a comment.

Comment (mknoedel) [inline .bowerrc]: For something else

File: bower.json (hunk 1 above)
Comment (navingator) [inline bower.json]: I think that bower is an essential tool. What is it again?

Comment (navingator) [inline bower.json]: Looked it up. NPM pretty much replaces this. We're good to go!

**Comment by mknoedel** [comment]
As a whole. I think this is pretty neat.

**Pull request merged**